### ISSUE TITLE
Tardis behavior when landing is blocked by another Tardis

### ISSUE THREAD

**Issue by Flexico** [DarkholmeTenk/TardisModIssues#587]
Ok, I've been fiddling around, and this is the behavior I've observed: If a tardis is headed to a particular set of co-ordinates and there is another tardis in the way, the travelling one just lands on top of the other one. That's fine and dandy. However, if a tardis is parked in a certain location (let's call it A) and it is then driven into drifting flight mode, and then another tardis (B) lands in that exact spot, when A tries to land, it does not materialize. However, the player is still able to leave A by way of the door, appearing outside B as if they had stepped out of that one.

If the player shift-right-clicks with the key of A, B will dematerialize. Shift-right-clicking again with the key will cause A to materialize in its place. If a player in B exits via the door, they end up outside A (b is not materialized). The tardises basically switched places.

I'm not sure what to make of the whole deal, but I thought you should know about it. And if you test this yourself, it's a good idea to change one tardis' appearance using the chameleon upgrade so they're easier to tell apart.
